priv/site/_includes/hero.slime

```diff
@@ -25,15 +25,10 @@ header.Banner
     = link @env, "Home", to: "/"
     = link @env, "Info", to: "/#info"
     marquee aria-hidden="true"
-      span All proceeds are for charity
-      span This is a carbon-neutral event
-      span 1 attendee = 1 tree
-      span All proceeds are for charity
-      span This is a carbon-neutral event
-      span 1 attendee = 1 tree
-      span All proceeds are for charity
-      span This is a carbon-neutral event
-      span 1 attendee = 1 tree
+      = for x <- 1..3 do
```

Comment:
Looks good but this will give out a warning due to `x` being unused. We can do `for _ <- 1..3` instead.